addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -348,5 +348,32 @@ def _get_summary_lines(self, templates):
             # We don't display deadlines when the user doesn't specify a plan_date
             if self.plan_date:
                 summary_line += f" ({format_date(self.env, template._get_date_deadline(self.plan_date))})"
+
+            # Add nested list for triggered and suggested next activities
+            next_activities = []
+            delay = f"{template.activity_type_id.delay_count} {dict(template.activity_type_id._fields['delay_unit']._description_selection(self.env))[template.activity_type_id.delay_unit]}"
+            delay_from = dict(template.activity_type_id._fields['delay_from']._description_selection(self.env))[template.activity_type_id.delay_from]
```

Comment:
it took me a while to understand this because here code readability is getting low so can we break this